docs/README.md

```diff
@@ -202,6 +202,12 @@ $ python3 scripts.develop/generate_api_key.py ${Your GitHub Username} local
 
 Now, `secrets/config.pforge-local-${Your GitHub Username}.yaml` should contain an api key.
 
+In addition, you will need to create a configuration file with the bakery information by copying the `bakeries/pangeo-ldeo-nsf-earthcube.pforge-local-cisasacstern.yaml` file to `bakeries/pangeo-ldeo-nsf-earthcube.local-${Your GitHub Username}.yaml`:
+
+```console
+$ cp bakeries/pangeo-ldeo-nsf-earthcube.pforge-local-cisasacstern.yaml bakeries/pangeo-ldeo-nsf-earthcube.local-${Your GitHub Username}.yaml
+```
```

Comment:
@cisaacstern, let me know if this isn't in the right place....

Reply:
@andersy005, this should probably under it's own subheading, right before **Start the Server** below.

Something like:

```
## Bakeries config

The last step before starting the app server is to setup bakeries config for your local app instance.

...
```

Reply:
Great! 👍